apps/web/composables/usePayPal/types.ts

```diff
@@ -0,0 +1,26 @@
+import { PayPalNamespace } from '@paypal/paypal-js';
+import {
+  PayPalCreateOrder,
+  PayPalApproveOrder,
+  PayPalExecutePayment,
+  PayPalExecuteParams,
+} from '@plentymarkets/plentymarkets-sdk/packages/api-client/server';
+
+export interface UsePayPalState {
+  PayPalScript: PayPalNamespace | null;
```

Comment:
```suggestion
  paypalScript: PayPalNamespace | null;
```
Is there a reason not to camel case here?